docs/Fields.md

```diff
@@ -495,10 +495,10 @@ With this configuration, `<ReferenceField>` wraps the user's name in a link to t
 </Admin>
 ```
 
-To change the link from the `<Edit>` page to the `<Show>` page, set the `linkType` prop to "show".
+To change the link from the `<Edit>` page to the `<Show>` page, set the `linkTo` prop to "show".
```

Comment:
Please name it `link` instead of `linkTo`

Reply:
Ok :)